projects/packages/status/src/class-modules.php

```diff
@@ -371,11 +388,12 @@ public function activate( $module, $exit = true, $redirect = true ) {
 			}
 		}
 
+		if ( ! $this->is_module( $module ) ) {
```

Comment:
why move this? Won't this be a regression for standalones using `activate()` - this check uses `validate_file()`.

Reply:
`validate_file` passes. it basically checks if the module is in the list of available module

Reply:
btw `validate_file` is a terrible function to be used there... it's basically doing a `! in_array(...`... maybe there was a time when we were actually checking for file paths... maybe we could just replace it with `in_array( $module, $this->get_available(), true )`

Reply:
wow yeah it really is terrible. The reason for all this file checking in Jetpack modules is because so much of it relies on the module header in the file being present. Lets just let this ride as-is for now unless you have to scratch this itch now :)